ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/b/DataModelWizardSpec.groovy

```diff
@@ -85,6 +89,7 @@ class DataModelWizardSpec extends AbstractModelCatalogueGebSpec {
         fill metadataApproved with '29/04/2016'
         fill 'metadata-namespace' with 'Namespace'
         fill 'metadata-organization' with 'Organization'
+        scroll ScrollDirection.UP
```

Comment:
you should be able to use just `scroll up` as `up` should be a constant in `Common`